tests/test_cameras.py

```diff
@@ -78,24 +109,24 @@ def test_camera(request, robot_type):
         camera.read()
     color_image = camera.read()
     async_color_image = camera.async_read()
-    print(
+    error_msg = (
         "max_pixel_difference between read() and async_read()",
         compute_max_pixel_difference(color_image, async_color_image),
     )
-    assert np.allclose(color_image, async_color_image, rtol=1e-5, atol=MAX_PIXEL_DIFFERENCE)
+    assert np.allclose(color_image, async_color_image, rtol=1e-5, atol=MAX_PIXEL_DIFFERENCE), error_msg
```

Comment:
Nit: Would maybe make rtol a constant like atol. Something like `PERCENTAGE_PIXEL_DIFFERENCE` (if I am understanding what it does correctly.

Reply:
I dont think this test works a real camera anyway. Without delving into details here, I will revisit this test when we set nightly tests on real robot devices.

```python
    # TODO(rcadene): properly set `rtol``
    assert np.allclose(color_image, async_color_image, rtol=1e-5, atol=MAX_PIXEL_DIFFERENCE), error_msg

```